content/zh-cn/blog/_posts/2023-12-18-kubernetes-1-29-feature-loadbalancer-ip-mode-alpha.md

```diff
@@ -0,0 +1,207 @@
+The "Proxy" value is intended to prevent kube-proxy from binding the load balancer's IP address 
+to the node in both ipvs and iptables modes. 
+Consequently, traffic is sent directly to the load balancer and then forwarded to the destination node. 
+The destination setting for forwarded packets varies depending on how the cloud provider's load balancer delivers traffic: 
+-->
+`.status.loadBalancer.ingress.ipMode`可选值为：`"VIP"`和`"Proxy"`。
+默认值为 `VIP`，即目标 IP 设置为负载均衡 IP 和端口并发送到节点的流量会被 kube-proxy 重定向到后端服务。这种方式保持 kube-proxy 现有行为模式。
+`Proxy`用于阻止 kube-proxy 在 ipvs 和 iptables 模式下绑定负载均衡 IP 地址到节点。
+此时，流量会直达负载均衡器然后被重定向到目标节点。转发数据包的目的值配置取决于云厂商的负载均衡器如何传输流量。
+
+<!-- 
+- If the traffic is delivered to the node then DNATed to the pod, the destination would be set to the node's IP and node port;
+- If the traffic is delivered directly to the pod, the destination would be set to the pod's IP and port. 
+-->
+- 如果流量被发送到节点然后通过目标地址转换（`DNAT`）的方式到达 Pod，目的值应当设置为节点和 IP 和端口；
+- 如果流量被直接转发到 Pod，目的值应当被设置为 Pod 的 IP 和端口。
+
+<!-- 
+## Usage 
+-->
+## 用法
+
+<!-- 
+Here are the necessary steps to enable this feature: 
+-->
+开启该特性的必要步骤：
+
+<!-- 
+- Download the [latest Kubernetes project](https://kubernetes.io/releases/download/) (version `v1.29.0` or later).
+- Enable the feature gate with the command line flag `--feature-gates=LoadBalancerIPMode=true` 
+on kube-proxy, kube-apiserver, and cloud-controller-manager.
+- For Services with `type: LoadBalancer`, set `ipMode` to the appropriate value. 
+This step is likely handled by your chosen cloud-controller-manager during the `EnsureLoadBalancer` process. 
+-->
+- 下载[Kubernetes最新版本](https://kubernetes.io/releases/download/) (`v1.29.0`或更新)。
+- 通过命令行参数`--feature-gates=LoadBalancerIPMode=true`在 kube-proxy、 kube-apiserver、和 cloud-controller-manager 开启特性门控。
+- 对`type: LoadBalancer`类型的服务, 将`ipMode`设置为合适的值。该步骤可能由你选择的 cloud-controller-manager 在`EnsureLoadBalancer`过程中操作。
+
+<!-- 
+## More information 
+-->
+## 更多信息
+
+- 阅读 [指定负载均衡器的IP模式](/docs/concepts/services-networking/service/#load-balancer-ip-mode)。
+- 阅读 [提案-1860](https://kep.k8s.io/1860) - [让Kubernetes感知到负载均衡器的行为](https://github.com/kubernetes/enhancements/tree/b103a6b0992439f996be4314caf3bf7b75652366/keps/sig-network/1860-kube-proxy-IP-node-binding#kep-1860-make-kubernetes-aware-of-the-loadbalancer-behaviour) _(sic)_。
+
+<!-- 
+## Getting involved 
+-->
+## 联系我们
+
+通过[Slack](https://slack.k8s.io/)联系我们: [#sig-network](https://kubernetes.slack.com/messages/sig-network), 
+或者通过[邮件列表](https://groups.google.com/forum/#!forum/kubernetes-sig-network)。
+
+<!-- 
+## Acknowledgments 
+-->
+## 特别鸣谢
+
+<!-- 
+Huge thanks to [@Sh4d1](https://github.com/Sh4d1) for the original KEP and initial implementation code. 
+I took over midway and completed the work. Similarly, immense gratitude to other contributors 
+who have assisted in the design, implementation, and review of this feature (alphabetical order): 
+-->
+非常感谢[@Sh4d1](https://github.com/Sh4d1)的原始提案和最初代码实现。
```

Comment:
```suggestion
非常感谢 [@Sh4d1](https://github.com/Sh4d1) 的原始提案和最初代码实现。
```

Reply:
pls check through. If no opinions, you just need to copy and paste the suggestions.